src/client/src/orangehrmCorporateDirectoryPlugin/components/SummaryCard.vue

```diff
@@ -107,12 +101,65 @@ export default {
       type: Boolean,
       default: false,
     },
+    employeeOnMobile: {
+      type: Boolean,
+      default: true,
+    },
   },
   emits: ['hide-details'],
   computed: {
     hasDefaultSlot() {
       return !!this.$slots.default;
     },
+    bodyClasses() {
+      return {
+        'orangehrm-directory-card-body': true,
+        'orangehrm-directory-card-body--change-body-height': !this
+          .showBackButton
```

Comment:
Conditions here are not clear. Please simplify. use computed property if needed.